libs/libssh/patches/0004-channels-Avoid-returning-SSH_AGAIN-from-ssh_channel_.patch

```diff
@@ -0,0 +1,41 @@
+From 6417f5a3cac8537ac6f6ff7fc1642dfaa0917fb4 Mon Sep 17 00:00:00 2001
+From: Jakub Jelen <jjelen@redhat.com>
+Date: Thu, 16 Apr 2020 11:13:34 +0200
+Subject: [PATCH] channels: Avoid returning SSH_AGAIN from
+ ssh_channel_poll_timeout()
+
```

Comment:
This addresses the issues mentioned in https://www.libssh.org/archive/libssh/2020-03/0000029.html and enables `libnetconf2` to work with `libssh-0.9.4`

@neheb Does this require a `PKG_RELEASE` bump for libssh?

Reply:
It's a code change so yes.

Reply:
Done